CHANGELOG.md

```diff
@@ -11,6 +11,9 @@ The following is a curated list of changes in the Enact agate module, newest cha
 - `agate/LabeledIconButton` max-width so that huge sized icon will not be cut off
 - `agate/Spinner` to pause the animation when `paused` prop is true
 
+### Removed
+- `agate/Item` last child border bottom style
+
```

Comment:
This is not API changes but virtual changes. It depends on each issue, but I think we don't need to update CHANGELOG.md for this PR.